storage/aptosdb/src/backup/restore_utils.rs

```diff
@@ -242,33 +242,42 @@ pub(crate) fn save_transactions_impl(
             first_version + idx as Version,
             txn,
             /*skip_index=*/ false,
-            batch,
+            &batch.transaction_db_batches,
         )?;
     }
-    ledger_store.put_transaction_infos(first_version, txn_infos, batch, batch)?;
-    event_store.put_events_multiple_versions(first_version, events, batch)?;
+    ledger_store.put_transaction_infos(
+        first_version,
+        txn_infos,
+        &batch.transaction_info_db_batches,
+        &batch.transaction_info_db_batches,
```

Comment:
the second one should be transaction accumulator batch?

Maybe you can also use another two functions, see the TODO here: https://github.com/aptos-labs/aptos-core/blob/main/storage/aptosdb/src/ledger_store/mod.rs#L286

Reply:
the 2nd function is not usable. might worth a separate PR to refactor this.